elide-async/src/main/java/com/yahoo/elide/async/service/AsyncQueryThread.java

```diff
@@ -57,11 +57,11 @@
 
     @Override
     public AsyncQueryResult call() throws NoHttpResponseException, URISyntaxException {
-         return processQuery();
+        return processQuery();
     }
 
     public AsyncQueryThread(AsyncQuery queryObj, User user, Elide elide, QueryRunner runner,
-            AsyncQueryDAO asyncQueryDao, String apiVersion, ResultStorageEngine resultStorageEngine) {
+                            AsyncQueryDAO asyncQueryDao, String apiVersion, ResultStorageEngine resultStorageEngine) {
```

Comment:
Instead of these many spaces to indent, 8 spaces from the beginning of the line will be fine to use here.